backend/gateway/meta/meta_test.go

```diff
@@ -118,6 +120,23 @@ func TestResourceNames(t *testing.T) {
 	}
 }
 
+func TestIsRedacted(t *testing.T) {
+	assert.True(t, IsRedacted(&authnv1.CallbackResponse{}))
+	assert.False(t, IsRedacted(&healthcheckv1.HealthcheckRequest{}))
+}
+
+func TestAPIBodyRedaction(t *testing.T) {
```

Comment:
Should we have a test for login response as well just to be sure the field isn't removed in the future?

Reply:
sure, i'll add tests to the auditing module itself to check this